### PR TITLE
issue #20 Create MergeSort.py

### PR DESCRIPTION
**Pull request by Nikunjpatel2411**
Fix: Issue #20 Hactoberfest2023 MergeSort.py

### CODE REVIEW

File: Sorting_Algorithm/PYTHON/MergeSort.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Python program for implementation of MergeSort
+# Author :- NikunjPatel
```
Comment (Agnik7) [inline Sorting_Algorithm/PYTHON/MergeSort.py:2]: It would be advisable to remove this line.

File: Sorting_Algorithm/PYTHON/MergeSort.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+def mergeSort(arr):
+	if len(arr) > 1:
+
```
Comment (Agnik7) [inline Sorting_Algorithm/PYTHON/MergeSort.py:6]: kindly remove these unnecessary spaces from this program. It is best to keep the code as compact as possible.

File: Sorting_Algorithm/PYTHON/MergeSort.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+# Driver Code
+if __name__ == '__main__':
+	arr = [12, 11, 13, 5, 6, 7]
+	print("Given array is")
+	printList(arr)
+	mergeSort(arr)
+	print("\nSorted array is ")
+	printList(arr)
+
+
```
Comment (Agnik7) [inline Sorting_Algorithm/PYTHON/MergeSort.py:62]: Using multiline comments, provide a sample example to show how the output will look like for the given array

**Review by Agnik7 — CHANGES_REQUESTED** [review]

File: Sorting_Algorithm/PYTHON/MergeSort.py (hunk 3 above)
Comment (Nikunjpatel2411) [reply Sorting_Algorithm/PYTHON/MergeSort.py:45]: Done

**Review by Nikunjpatel2411 — COMMENTED** [review]
Updated MergeSort.py check it #20

**Review by Nikunjpatel2411 — COMMENTED** [review]
Updated MergeSort.py check it #20

**Comment by Nikunjpatel2411** [comment]
Now , 
Check it

**Review by Agnik7 — APPROVED** [review]

**Pull request merged**